### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by pandabadger**
Fix some typos

### CODE REVIEW

File: docs/core/DelegationManager.md [hunk 1]
```diff
@@ -375,7 +375,7 @@ Called by the `EigenPodManager` when a Staker's shares decrease. This method is
 * `EigenPod.verifyAndProcessWithdrawals`
 
 *Effects*: If the Staker in question is delegated to an Operator, the Operator's delegated balance for the `strategy` is decreased by `shares`
-* This method is a no-op if the Staker is not delegated an an Operator.
+* This method is a no-op if the Staker is not delegated as an Operator.
```
Comment (ChaoticWalrus) [inline docs/core/DelegationManager.md:378]: should be 'to' an operator

File: docs/core/EigenPod.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ We will take a brief aside to explain a simpler part of the protocol, partial wi
...
 
-Currently, users can submit partial withdrawal proofs one at a time, at a cost of around 30-40k gas per proof.  This is highly inefficient as partial withdrawals are often nominal amounts for which an expensive proof transaction each time is not feasible for our users.  The solution is to use succinct zk proving solution generate a single proof for multiple withdrawals, which can be verified for a fixed cost of around 300k gas.  This system and associated integration are currently under development.  
+Currently, users can submit partial withdrawal proofs one at a time, at a cost of around 30-40k gas per proof.  This is highly inefficient as partial withdrawals are often nominal amounts for which an expensive proof transaction each time is not feasible for our users.  The solution is to use a succinct zk proving solution to generate a single proof for multiple withdrawals, which can be verified for a fixed cost of around 300k gas.  This system and associated integration are currently under development.
```
Comment (ChaoticWalrus) [inline docs/core/EigenPod.md:61]: the added 'a' is extraneous. Succinct is an entity here

**Review by ChaoticWalrus — CHANGES_REQUESTED** [review]
Thanks for the PR. noted some issues with the suggestions

**Comment by pandabadger** [comment]
updated

**Comment by ChaoticWalrus** [comment]
> updated

LGTM, ty ty 😄

**Pull request merged**